vtk/meta.yaml

```diff
@@ -12,12 +12,14 @@ source:
   fn: VTK-5.10.1.win-amd64-py2.7.exe         [win64 and py27]
 
 build:
-  number: 1                                  [linux]
+  number: 2                                  [linux]
 
 requirements:
   build:
     - cmake                                  [linux]
     - python
+    - qt
+    - pyqt
```

Comment:
I'm just saying that, as far as conda is involved, adding `pyqt` is enough because that dependency will pull `qt` by default.